### PR TITLE
MAINT: Update to the new API of sMRIPrep

### PR DESCRIPTION
**Pull request by oesteban**
Replaces #1827 because I need extra flexibility to solve issues (and I believe @adelavega will appreciate that I take care of this).

### CODE REVIEW

**Comment by oesteban** [comment]
@adelavega you'd be very welcome to add your name to the contributors' list (which is ordered alphabetically by lastname):

https://github.com/poldracklab/fmriprep/blob/7bc943ef0739eba1e1cde4ff390d505d7c2256ef/.maint/contributors.json

File: fmriprep/workflows/base.py [hunk 1]
```diff
@@ -619,26 +619,26 @@ def init_single_subject_wf(
...
               ('outputnode.subjects_dir', 'inputnode.subjects_dir'),
               ('outputnode.subject_id', 'inputnode.subject_id'),
-              ('outputnode.t1_2_fsnative_forward_transform',
-               'inputnode.t1_2_fsnative_forward_transform'),
-              ('outputnode.t1_2_fsnative_reverse_transform',
-               'inputnode.t1_2_fsnative_reverse_transform')]),
+              ('outputnode.t1w2fsnative_xfm',
+               'inputnode.t1w2fsnative_xfm'),
+              ('outputnode.fsnative2t1w_xfm',
+               'inputnode.fsnative2t1w_xfm')]),
```
Comment (effigies) [inline fmriprep/workflows/base.py]: These two look like they could each go on one line now.

File: fmriprep/workflows/bold/base.py [hunk 2]
```diff
@@ -199,16 +199,16 @@ def init_func_preproc_wf(
...
             Bias-corrected structural template image
-        t1_brain
-            Skull-stripped ``t1_preproc``
-        t1_mask
+        t1w_brain
+            Skull-stripped ``t1w_preproc``
+        t1w_mask
             Mask of the skull-stripped template image
-        t1_seg
+        t1w_dseg
```
Comment (mgxd) [inline fmriprep/workflows/bold/base.py]: `t1w_aseg` and `t1w_aparc` are missing here

Comment (oesteban) [reply fmriprep/workflows/bold/base.py]: sorry, I missed this one. I'll try to get at this asap

Comment (oesteban) [reply fmriprep/workflows/bold/base.py]: done (7e5b2f1288f53ed54413d997a615a91a999a3597)

**Pull request merged**